apps/send/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "send",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "devDependencies": {
+    "@types/node": "20.5.9",
+    "@types/react": "18.2.21",
+    "@types/react-dom": "18.2.7",
+    "autoprefixer": "10.4.15",
```

Comment:
For the goal of this activity, do we need this dependency?

Reply:
also for those, better to have them even if is a basic installation. Those small modules are useful while developing

Reply:
Same here, the comment is pointing only to `autoprefixer`

Reply:
For the autoprefixer, is needed for any .css style to be applied in the project. This will minimize all the code from the css and will add vendor prefixes automatically part like this example:

If we wrote this:

.App {
  display: flex;
  flex-direction: row;
  align-items: center;
}

with the autoprefixer it will become like this:

.App {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-orient: horizontal;
  -webkit-box-direction: normal;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

without the autoprefixer we need to write each time all those vendor prefixes due to different user's browsers, this could create human errors and a lot of time spending. Could be removed in this stage, but I think that 100% we'll need to install again when we'll start working on any website.